### PR TITLE
MA-1401: Generate alerts / tickets when account regisration fails with ESB / IT

### PR DESCRIPTION
**Pull request by chandramishra**
**Purpose:**   Improve error message
For more details Please refer this link : https://mountdiablo.jira.com/browse/MA-1401?filter=-1
**Changes:**  aws-services-organizations micro services
**Integration testing/functional testing:**
1) Create account when billing system not reachable
2) Create account with wrong api & secret key
3) Create account with existing email id
‌
**Unit Testing**
Not Available

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -133,11 +133,12 @@ function checkCreationStatus(organizations, input, callback) {
         sleep.sleep(5); //sleep for 5 seconds
         checkCreationStatus(organizations, input, callback);
       }
-      else if (data.CreateAccountStatus.State == "SUCCEEDED") {
-        callback(null, data);
-      }
+      //else if (data.CreateAccountStatus.State == "SUCCEEDED") {
```
Comment (RajkumarShivage) [inline src/index.js]: if these are not required, remove these lines.

**Review by RajkumarShivage — APPROVED** [review]
LGTM

**Review by SonalOjha — COMMENTED** [review]
The changes looks like we need to update the ticket with the relevant data rather than custom message.

**Comment by chandramishra** [comment]
This file modified just for return actual message to next flow so that we can send/or log exact root cause (why account creation failed) message

**Review by SonalOjha — APPROVED** [review]

**Pull request merged**